lametro/models.py

```diff
@@ -563,3 +563,115 @@ class Meta:
 
     guid = models.CharField(max_length=256)
     name = models.CharField(max_length=256, unique=True)
+
+
+class BillPacket(models.Model):
+
+    bill = models.OneToOneField(LAMetroBill,
+                                related_name='packet',
+                                on_delete=models.CASCADE)
+    updated_at = models.DateTimeField(auto_now=True)
+    url = models.URLField()
+    ready = models.BooleanField(default=False)
+
+    def save(self, *args, **kwargs):
+
+        self._merge_docs()
+
+        self.url = settings.MERGER_BASE_URL + '/document/' + self.bill.slug
+
+        response = requests.head(self.url)
+
+        super().save(*args, **kwargs)
+
+    def is_ready(self):
```

Comment:
Where is this used?

Reply:
https://github.com/datamade/la-metro-councilmatic/blob/2.5/lametro/templates/lametro/event.html#L86

Reply:
https://github.com/datamade/la-metro-councilmatic/blob/2.5/lametro/templates/lametro/legislation.html